packages/teams-js/src/internal/telemetry.ts

```diff
@@ -217,11 +222,37 @@ export enum ApiName {
   PublicAPIs_ShareDeepLink = 'shareDeepLink',
   RegisterHandler = 'registerHandler',
   RegisterOnThemeChangeHandler = 'registerOnThemeChangeHandler',
-  Settings_Remove_Failure = 'settings.remove.failure',
-  Settings_Remove_Success = 'settings.remove.success',
-  Settings_Save_Failure = 'settings.save.failure',
-  Settings_Save_Success = 'settings.save.success',
+  Search_CloseSearch = 'search.closeSearch',
+  Search_RegisterOnChangeHandler = 'search.registerOnChangeHandler',
+  Search_RegisterOnClosedHandler = 'search.registerOnClosedHandler',
+  Search_RegisterOnExecutedHandler = 'search.registerOnExecutedHandler',
+  Search_UnregisterHandlers = 'search.unregisterHandlers',
+  SecondaryBrowser_OpenUrl = 'secondaryBrowser.openUrl',
+  Settings_GetSettings = 'settings.getSettings',
+  Settings_RegisterOnRemoveHandler = 'settings.registerOnRemoveHandler',
+  Settings_RegisterOnSaveHandler = 'settings.registerOnSaveHandler',
+  Settings_SetSettings = 'settings.setSettings',
+  Settings_SetValidityState = 'settings.setValidityState',
+  StageView_Open = 'stageView.open',
   Tasks_StartTask = 'tasks.startTask',
   Tasks_SubmitTask = 'tasks.submitTask',
   Tasks_UpdateTask = 'tasks.updateTask',
+  VideoEffects_MediaStream_RegisterForVideoFrame = 'videoEffects.mediaStream.registerForVideoFrame',
+  VideoEffects_NotifySelectedVideoEffectChanged = 'videoEffects.notifySelectedVideoEffectChanged',
+  VideoEffects_NotifyError = 'videoEffects.notifyError',
+  VideoEffects_NotifyVideoFrameProcessed = 'videoEffects.notifyVideoFrameProcessed',
+  VideoEffects_RegisterEffectParameterChangeHandler = 'videoEffects.registerEffectParameterChangeHandler',
+  VideoEffects_RegisterForVideoEffect = 'videoEffects.registerForVideoEffect',
+  VideoEffects_RegisterForVideoFrame = 'videoEffects.registerForVideoFrame',
+  VideoEffects_RegisterSetFrameProcessTimeLimitHandler = 'videoEffects.setFrameProcessTimeLimitHandler',
+  VideoEffects_RegisterStartVideoExtensibilityVideoStreamHandler = 'videoEffects.startVideoExtensibilityVideoStreamHandler',
+  VideoEffects_RegisterForVideoBufferHandler = 'videoEffects.registerForVideoBufferHandler',
+  VideoPerformanceMonitor_Constructor = 'videoPerformanceMonitor.performanceDataGenerated',
```

Comment:
@TrevorJoelHarris Based on discussion with @WengHe , videoPerformanceMonitor doesn't have to send API telemetry but since we would like to treat different APIs with no difference, I would like to treat videoPerformanceMonitor as a 'capability' and put some APIs inside of telemetry list. Though apiVersionTag is composed and used for videoPerformanceMonitor's APIs, they actually are useless because they would not be sent to Kusto database.

The core concern is just to make sure we could force each API to be labelled as apiVersionTag so that we don't have two different functions and use either one of them incorrectly.